tests/conftest.py

```diff
@@ -508,6 +519,12 @@ def product_list(product_type, category):
             ),
         ]
     )
+    sku = 0
+    for product in products:
+        ProductVariant.objects.create(
+            product=product, sku=sku, track_inventory=True, quantity=100
+        )
+        sku = sku + 1
```

Comment:
This could generate a problem with db because sku is unique

Reply:
Are you sure? We have something like `123` or `555` in other places.

Reply:
Try to use fixtures `product` and `unavailable_product_with_variant` in the same test ;)
If we don't use tools like faker we should keep in mind that unique fields shouldn't be duplicated (like `uuid4()`)